Dockerfile

```
@@ -16,4 +16,4 @@ CMD ["fpindex", "--dir", "/var/lib/acoustid-index", "--address", "0.0.0.0", "--p
 # Final stage with binary copied in
 FROM base AS final
 
-ADD zig-out/bin/fpindex /usr/bin
+ADD --chmod +x zig-out/bin/fpindex /usr/bin
```

Hadolint (2.14.0)

[error] unexpected '-'
expecting '\', at least one space, or only the --checksum, --chown, --chmod, --link, --exclude flags or the src and dest paths

(DL1000)